.github/README.md

```diff
@@ -17,76 +17,66 @@
 
 ## Introduction
 
-AzerothCore is an open-source game server application and framework designed for hosting massively multiplayer online role-playing games (MMORPGs). It is based on the popular MMORPG World of Warcraft (WoW) and seeks to recreate the gameplay experience of the original game from patch 3.3.5a.
+AzerothCore is a community-driven, open-source game server framework designed to host massive multiplayer online role-playing games (MMORPGs). This project is based on World of Warcraft (WoW) and aims to recreate the gameplay experience of the Wrath of the Lich King (WotLK) expansion, specifically patch 3.3.5a.
 
-The original code is based on MaNGOS, TrinityCore, and SunwellCore and has since then had extensive development to improve stability, in-game mechanics, and modularity to the game. AC has also grown into a community-driven project with a significant number of contributors and developers. It is written in C++ and provides a solid foundation for creating private servers that mimic the mechanics and behavior of the official WoW servers.
+Originally derived from MaNGOS, TrinityCore, and SunwellCore, AzerothCore has developed extensively to improve stability, in-game mechanics, and modularity. The project has become a reliable foundation for creating private servers replicating the official WoW experience while fostering a collaborative environment for developers and enthusiasts.
```

Comment:
wording of the last sentence is worse than before imo

Reply:
I disagree because I think it gives more of a "community" feeling. But that's me